project_development_tips.md

```diff
@@ -0,0 +1,119 @@
+
+Some of these suggestions might seem somewhat unnecessary, but it is important to model what a project can do to manage its smart contracts the best so that hopefully all of the projects follow suit. It will also make all the Flow smart contracts that much safer and easier to use.
+
+This also assumes standard software design best practices also apply. Indeed, many of these suggestions are more general software design best practices, but there may be others that are assumed but not included here.
+
+### Implementing These Practices
+
+This document serves as mostly an outline of best practices the projects should follow. Engineering leadership needs to figure out concrete steps for actually implementing them at their respective companies including detailed documents, education sessions, and workshops with all managers and Cadence developers.
+
+## **Design Process**
+
+- Ensure that there is a single person ultimately responsible for the project.
+    - It is this person’s responsibility to lead design discussions with product managers and the community, write most of the code and tests, solicit reviews, make requested changes and make sure the project gets completed in a timely manner.
+    - Even if the project is relatively easy, the owner should be someone who understands Cadence well and who keeps in touch with what is happening in the Cadence world. Examples:
+        - Responsible project maintainers should contribute to discussions about important proposals (new cadence features, standard smart contracts, metadata, etc) and generally be aware about evolving best practices and anti-pattern understandings.
+        - They should regularly re-evaluate the code of the project to see if it needs to be updated to conform to standards and new versions of Cadence.
+        - The best projects should be active in the development of the protocol and language.
+    - The owner should also understand how to sign transactions with the CLI to upgrade the smart contract, run admin transactions, deploy new contracts, etc. 
+- The owner should identify a secondary owner who doesn’t hold any direct responsibility, but still has a clear understanding of the code and requirements so they can give good feedback, perform effective reviews, and make changes where needed. In case the owner is not available or leaves the project, this person can also take over easily so there is never a time when there is nobody who understands the smart contracts well.
+- If there isn’t already an open source repo, the owner should create a public GitHub repo from [the open source template](https://github.com/onflow/open-source-template) and make sure all of this is set up with some initial documentation for what the repo is for before any code is written.
+- The owner should create a design document that lays out the intended high level design and architecture of the smart contract and some basic user stories, as well as any questions that still need to be answered about it.
```

Comment:
Probably we should provide some examples about this point if we really want people to do it, but also probably those examples belong outside this document?

Reply:
we could link to an example, like your switchboard document or something like that. Are you ok with that?

Reply:
Yeah absolutely! Maybe we can take a look at it before using it as an example since I did it literally on my second week at dapper, but I'll be honoured!